alembic/versions/dc382319dd84_seed_db.py

```diff
@@ -0,0 +1,64 @@
+        {'id':10,'name':'Kenai','plant_type':'Bear Paw','image':'https://cdn.shopify.com/s/files/1/2391/9491/products/image_ff75aeba-99e0-4791-b21b-74580ff9303b_1024x1024.jpg?v=1571710608','lighting':'Full Sun','water_frequency':7,'root _depth':4,'annual':False},
+        {'id':11,'name':'Ken','plant_type':'Bamboo','image':'https://cdn.pixabay.com/photo/2020/05/24/11/53/bamboo-5213909_960_720.jpg','lighting':'Indirect Sun','water_frequency':4,'root _depth':36,'annual':False},
+        {'id':12,'name':'Mary','plant_type':'Raspberry','image':'https://www.publicdomainpictures.net/pictures/10000/velka/1-1248158051Ix2h.jpg','lighting':'Full Sun','water_frequency':4,'harvest_time':456,'root _depth':20,'annual':False},
+        {'id':13,'name':'Jane','plant_type':'Blackberry','image':'https://www.publicdomainpictures.net/pictures/10000/velka/1-1248160910fXMI.jpg','lighting':'Full Sun','water_frequency':4,'harvest_time':730,'root _depth':14,'annual':False},
+        {'id':14,'name':'Randy','plant_type':'Strawberry','image':'https://www.picserver.org/pictures/strawberries01-lg.jpg','lighting':'Full Sun','water_frequency':4,'harvest_time':152,'root _depth':12,'annual':False},
+        {'id':15,'name':'Howdi','plant_type':'Avocado','image':'https://c1.peakpx.com/wallpaper/736/669/216/appetite-avacado-avo-avocado-wallpaper-preview.jpg','lighting':'Full Sun','water_frequency':7,'harvest_time':3285,'root _depth':12,'annual':False},
+        {'id':16,'name':'Russel','plant_type':'Cilantro','image':'https://storage.needpix.com/rsynced_images/cilantro-1287301_1280.jpg','lighting':'Full Sun/Light Shade','water_frequency':3,'harvest_time':70,'root _depth':18,'annual':True},
+        {'id':17,'name':'Todd','plant_type':'Lemon','image':'https://p0.pikrepo.com/preview/784/509/two-lemons-and-1-slice-of-lemon.jpg','lighting':'Full Sun','water_frequency':4,'harvest_time':365,'root _depth':24,'annual':False},
+        {'id':18,'name':'Leslie','plant_type':'Lime','image':'https://i2.pickpik.com/photos/197/226/505/sliced-lime-fruit-lime-slice-preview.jpg','lighting':'Full Sun','water_frequency':4,'harvest_time':121,'root _depth':24,'annual':False},
+        {'id':19,'name':'Henri','plant_type':'Grapes','image':'https://p0.pikist.com/photos/670/14/autumn-grapes-grape-vine-fruit-winegrowing-grapevine-blue-grapes-wine-harvest.jpg','lighting':'Full Sun','water_frequency':7,'harvest_time':1095,'root _depth':36,'annual':False},
+        {'id':20,'name':'Jennifer','plant_type':'Asparagus','image':'https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcS2HpJaaQQPpaDrolZY-cg9owfb4z9oy2eZ-A&usqp=CAU','lighting':'Full Sun','water_frequency':21,'harvest_time':730,'root _depth':10,'annual':False},
+        {'id':21,'name':'Sunshine','plant_type':'Daffodil','image':'https://p0.pikist.com/photos/246/87/daffodils-spring-flowers-daffodil-bloom-nature-yellow-petals-garden.jpg','lighting':'Full Sun','water_frequency':7,'root _depth':6,'annual':False},
```

Comment:
One or two of these images need to be replaced as they download the photo when the URL is opened, rather than opening the page on the browser. I will look into which ones need to be replaced later today.